manifests/setup/fluent.io/fluent-operator-clusterRole.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/name: fluent-operator
+  name: kubesphere:operator:fluent-operator
```

Comment:
`kubesphere:operator:fluent-operator` => `fluent-operator` is ok?

Reply:
fix these done